dvc/config_schema.py

```diff
@@ -145,6 +145,7 @@ class RelPath(str):
                     "session_token": str,
                     Optional("listobjects", default=False): Bool,  # obsoleted
                     Optional("use_ssl", default=True): Bool,
+                    Optional("ssl_verify", default="true"): str,
```

Comment:
If I do `dvc remote modify something ssl_verify true` or `dvc remote modify something ssl_verify false`, how will it automatically coerce these to bool values rather than paths? I might be missing something but it looks like it will always use these as strings, and even I say `"false"` it will interpret it either as a path or a true value.

Reply:
It could probably be done with something like `supported_cache_type` that we use above, but it really shows that we are probably better off not combining bool and str in this option, as even the name is not really friendly for the `path` case. As @skshetry mentioned above, we could introduce that in a separate PR later, for now keeping `ssl_verify` only as Bool

Reply:
Agreed

Reply:
Suggestion, I could add an option to enter a path as a str. Boto can handle both on the verify param, so we only nee to discuss which one had priority above the other?

Reply:
@isidentical I've just made a function that parses this parameter. Also tested all cases on this function